packet/utils.py

```diff
@@ -85,3 +85,10 @@ def wrapped_function(*args, **kwargs):
         return func(*args, **kwargs)
 
     return wrapped_function
+
+def notify_slack(name: str):
```

Comment:
Looks like you renamed `shower` to `notify_slack`, please fix the code so that it builds

Reply:
I adjusted the import statement to resolve this.